### PR TITLE
Spike: Use vanilla Bootstrap as ui framework

### PR DESCRIPTION
**Pull request by mxinden**
This is a quick spike testing out Bootstrap as our UI framework.

@stuartnelson3  @w0rm What do you think?

This is not focussed on design but rather technical usability.

### CODE REVIEW

File: ui/app/index.html [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-    var app = Elm.Main.embed(document.body);
-  </script>
+    <!-- Removes splash and starts elm app. -->
+    <script type="text/javascript">
+     var app = Elm.Main.embed(document.body);
+    </script>
 
+
+    <!-- Bootstrap requirements -->
+    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
```
Comment (w0rm) [inline ui/app/index.html]: We should only use css from bootstrap. Js won't work with Elm.

Comment (mxinden) [inline ui/app/index.html]: Done

**Comment by stuartnelson3** [comment]
Looks cool to me. 

Eventually we'll want to serve up bootstrap from alertmanager itself. We could even embed it (and script.js) in the index.html when we build it. But, that's for a different PR.

**Comment by mxinden** [comment]
Cool. Waiting for @w0rm final words. Then I will merge the PR if there are no other objections.

**Pull request merged**